style.css

```diff
@@ -0,0 +1,93 @@
+.heading{
+    margin-left: 40%;
+    text-align: center;
+    width: 200px;
+    height: 45px;
+    background-color: white;
+    border: 2px solid orange;
+}
+header{
+    background-color: darkslategrey;
+    padding: 10px;
+}
+h2{
+    font-size: 20px;
+    margin: 2px;
+}
+small{
+    margin-top: 2px;
+    word-spacing: 0%;
+}
+.lin1{
+    margin-top: 20px;
+}
+#lin11{
+    margin-left: 20px;
+}
+#lin12{
+   margin-left: 800px;

```

Comment:
this is very huge value for margin to have  .Please read more on the CSS concepts